frontend/pages/policies/ManagePoliciesPage/components/AddPolicyModal/AddPolicyModal.tsx

```diff
@@ -36,9 +36,11 @@ const AddPolicyModal = ({
     setLastEditedQueryCritical,
     setLastEditedQueryPlatform,
     setPolicyTeamId,
+    setDefaultPolicy,
   } = useContext(PolicyContext);
 
   const onAddPolicy = (selectedPolicy: IPolicyNew) => {
+    setDefaultPolicy(true);
```

Comment:
When adding a default policy, new defaultPolicy key is set to true, which is used to override the check for policy platform